driver/driver.py

```diff
@@ -0,0 +1,181 @@
+import os
+import yaml
+import logging
+import shutil
+from pathlib import Path
+from datetime import datetime, timedelta, timezone
+from multiprocessing import Pool
+import subprocess
+import wxflow
+from wxflow import Logger, Jinja
+from wxflow.configuration import cast_as_dtype
+
+def generate_eva_config(template_path: str, output_path: str, context: dict):
```

Comment:
Do we really need this function? isn't this a duplication of something in wxflow?

Reply:
It is not a replication of something in wxflow, it uses wxflow's capabilities. If you dont think it needs to be a function I will just add it back into the main driver function.